### ISSUE TITLE
Pascal-50S and FOIL

### ISSUE THREAD

**Issue by joeyz0z** [aimagelab/pacscore#4]
Thank you very much for your excellent work! I want to reproduce the experiments on Pascal-50S and FOIL datasets. Could you provide the evaluation code and download link for these two datasets or give me some guidance? I would appreciate it if you could take the time to reply to me.

**Comment by sarasarto** [comment]
Regarding the reproducibility of the experiments conducted on Pascal-50S, we have referenced the issue on the CLIPScore GitHub, which can be found at this link https://github.com/jmhessel/clipscore/issues/4 . For accessing the Pascal-50S dataset, you can use the following download link: https://vrama91.github.io/cider/.
*Important Note:* Please be aware that for our evaluations, we randomly sample five reference captions (among the 48 provided by the dataset) and average the results over five different draws.

The download link for the FOIL dataset is https://foilunitn.github.io/. 
*Important Note:* the FOIL dataset is composed of image-caption pairs from the COCO dataset, therefore, for a fair comparison, we take the subset of the validation set that does not overlap with the portion of COCO used to finetune our model thus obtaining 8k images, each associated with a foil-correct textual pair.

**Comment by joeyz0z** [comment]
Thanks for your reply! It helps me a lot!
